src/NServiceBus.AcceptanceTests/EndpointTemplates/Requires.cs

```diff
@@ -2,7 +2,8 @@
 {
     using NUnit.Framework;
 
-    static class Requires
+    // ReSharper disable once PartialTypeWithSinglePart
+    static partial class Requires
```

Comment:
Visual Studio also has a non-blocking error doesn't that shows up related to this. Should also wrap this in

`#pragma warning disable RECS0001`/`#pragma warning restore RECS0001` ?

Reply:
Hmm... didn't see this in my VS instance. Not even with release compilation

Reply:
I'm seeing it on mine when I have the file open, but maybe it's coming from an extension I have installed?

Reply:
This is what I have 

![image](https://user-images.githubusercontent.com/174258/30986667-7ce4ae96-a494-11e7-91f7-892cb2377e83.png)

Reply:
Let me figure out where's coming from then.

Reply:
Ah, that comes from http://vsrefactoringessentials.com/ which I have installed.

I'd say we don't need to worry about adding the #pragmas then.